djangocms_blog/admin.py

```diff
@@ -30,6 +31,59 @@ class EnhancedModelAdminMixin(object):
         pass
 
 
+# Bulk actions for post admin
+def make_published(modeladmin, request, queryset):
+    """ Bulk action to mark selected posts as published. If
+        the date_published field is empty the current time is
+        saved as date_published.
+    """
+    for post in queryset:
```

Comment:
Well, maybe I did not think that through thoroughly... Will use a ``queryset.update``.